tests/components/zerproc/test_config_flow.py

```diff
@@ -0,0 +1,30 @@
+"""Test the zerproc config flow."""
+from asynctest import patch
+
+from homeassistant.components.zerproc.config_flow import _async_has_devices
+
+
+class MockException(Exception):
+    """Mock exception class."""
+
+
+async def test_has_devices(hass):
+    """Test we get the form."""
+    with patch(
+        "homeassistant.components.zerproc.config_flow.pyzerproc.discover",
+        return_value=[],
+    ):
+        assert await _async_has_devices(hass) is False
```

Comment:
Please use the core interfaces to start the tests. Here use `hass.config_entries.flow.async_init` and start the user step.

See eg:
https://github.com/home-assistant/core/blob/823a44cd231c866868ef82c0b3d6cf04475a6539/script/scaffold/templates/config_flow/tests/test_config_flow.py#L9-L16